DomainContent/AvatarStore/attachmentItemScript.js

```diff
@@ -0,0 +1,156 @@
+                _supportedJoints.push("Right".concat(baseJoint));
+            } else {
+                _supportedJoints.push(_attachmentData.joint);
+            }
+
+            isAttached = _attachmentData.attached;
+
+            if (Entities.getNestableType(properties.parentID) !== "avatar" && !isAttached) {
+                messageChannel = MESSAGE_CHANNEL_BASE + properties.parentID;
+                Messages.subscribe(messageChannel);
+            }
+        },
+        startNearGrab: function(entityID, args) {
+            if (firstGrab) {
+                var properties = Entities.getEntityProperties(entityID);  
+                if (!properties.visible) {
+                    Entities.editEntity(entityID, {visible: true});
+                } 
+                firstGrab = false;
+            }
+        },
+            
+        releaseGrab: function(entityID, args) {
+            var hand = args[0] === "left" ? 0 : 1;
+            var properties = Entities.getEntityProperties(entityID);
+
+            if (Entities.getNestableType(properties.parentID) === "entity") {
+                Messages.sendMessage(messageChannel, "Removed Item :" + entityID);
+                Messages.unsubscribe(messageChannel); 
+                Entities.editEntity(entityID, {parentID: EMPTY_PARENT_ID});
+            }
+
+            var userData = properties.userData;
+            var position = properties.position; 
+            var attachmentData = JSON.parse(userData).Attachment;
+            isAttached = attachmentData.attached;
+
+            if (!isAttached) {
+                _supportedJoints.forEach(function(joint) {
+                    var jointPosition = MyAvatar.getJointPosition(joint);
+                    if (Vec3.distance(position, jointPosition) <= ATTACH_DISTANCE) {
+                        var newEntityProperties = Entities.getEntityProperties(_entityID);
+                        touchJSONUserData(newEntityProperties, function(userData) {
+                            userData.Attachment.attached = true;
+                        });
+                        Entities.editEntity(_entityID, {
+                            parentID: MyAvatar.sessionUUID,
+                            parentJointIndex: MyAvatar.getJointIndex(joint),
+                            userData: newEntityProperties.userData,
+                            lifetime: -1
+                        });
+                        if (ATTACH_SOUND.downloaded) {
+                            Audio.playSound(ATTACH_SOUND, {
+                                position: MyAvatar.position,
+                                volume: AUDIO_VOLUME_LEVEL,
+                                localOnly: true
+                            });
+                        }
+                        Controller.triggerHapticPulse(TRIGGER_INTENSITY, TRIGGER_TIME, hand);
+                    }
+                }); 
+            } else if ( isAttached) {
```

Comment:
there's an unnecessary  space on the left of isAttached  `( isAttached)` -> `(isAttached)`